src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -378,7 +390,7 @@ export default class PanoImageRenderer extends Component {
 	}
 
 	renderWithQuaternion(quaternion, fieldOfView) {
-		if (!this.hasRenderingContext()) {
+		if (!this.isImageLoaded() || !this.hasRenderingContext()) {
```

Comment:
The **isImageLoaded** method seems like no longer belongs to the **PanoImageRenderer**.

How about removing isImageLoaded method from PanoImageRenderer and checking loading status with method on VideoLoader and ImageLoader?

like 
```js
this._imageLoader.isContentReady();
this._videoLoader.isContentReady();
```

Reply:
@happyhj I got it!